src/main/scala/org/ergoplatform/network/ModeFeature.scala

```diff
@@ -0,0 +1,73 @@
+package org.ergoplatform.network
+
+import com.google.common.primitives.Ints
+import org.ergoplatform.nodeView.state.StateType
+import org.ergoplatform.settings.NodeConfigurationSettings
+import scorex.core.network.PeerFeature
+import scorex.core.network.PeerFeature.Id
+import scorex.core.serialization.Serializer
+import scorex.core.utils
+
+import scala.util.Try
+
+case class ModeFeature(stateType: StateType,
+                       verifyingTransactions: Boolean,
+                       popowBootstrapping: Boolean,
```

Comment:
May be it's better to combine `popowBootstrapping` and `popowSuffix` to single parameter `popowSuffix: Option[Int]`

Reply:
done